dymos/transcriptions/pseudospectral/components/gauss_lobatto_interleave_comp.py

```diff
@@ -77,19 +77,19 @@ def add_var(self, name, shape, units, disc_src, col_src):
         size = np.prod(shape)
 
         self._varnames[name] = {}
-        self._varnames[name]['disc'] = f'disc_values:{name}'
+        self._varnames[name]['state_disc'] = f'disc_values:{name}'
         self._varnames[name]['col'] = f'col_values:{name}'
         self._varnames[name]['all'] = f'all_values:{name}'
 
         # Check to see if the given disc source has already been used
```

Comment:
Really, really minor thing but in the comment do you want to be explicit about it being a state disc? In comments about control discs, it was explicit